containers-ordered-strict/src/Map/Ord.hs

```diff
@@ -35,12 +35,16 @@ type Key = Ord
 
 lookupDefault :: Key k => v -> k -> Map k v -> v
 lookupDefault = M.findWithDefault
+{-# INLINE lookupDefault #-}
```

Comment:
I think we should add `-fexpose-all-unfoldings` to the `containers-ordered-strict` package as well.